lib/krb5-wrapper.c

```diff
@@ -601,11 +601,11 @@ krb5_init_server_cred(struct smb2_server *server, struct smb2_context *smb2, con
                 /* do constrained delegation */
                 cred_usage = GSS_C_BOTH;
 #ifndef __APPLE__
-                /* get a proxy ticket when user is known in session reply.  note that this
-                 * isnt absolutely required if the accept returns a delegatable cred but it
-                 * does help find issues
+                /* set this non-0 to also get proxy ticket when user is known in
+                 * session reply.  note that this isnt required if the accept returns
+                 * a delegatable cred but it might help find issues (or cause some)
                  */
-                auth_data->s4u2self = 1;
+                auth_data->s4u2self = 0;
```

Comment:
Note - if a delegatable cred is gotten there is no need to also do an s4u2self request.  In fact, the code I made to do that was a bit wrong, and ended up doing a service -> remote-service check which worked by mistake on my AD DC because of extra delegation I had setup.  It wants to a user->remote-service check instead which I will work on at some point so plz leave the code in here?